### ISSUE TITLE
Typescript compilation error with React 18

### ISSUE THREAD

**Issue by nuarhu** [headzoo/react-moment#152]
Hello,

while upgrading React and react-moment, I get the following compiler warning:

```
node_modules/react-moment/dist/index.d.ts:2:37 - error TS2305: Module '"react"' has no exported member 'SFC'.
2 import { Component, ComponentClass, SFC, CSSProperties } from 'react';
                                      ~~~
Found 1 error in node_modules/react-moment/dist/index.d.ts:2
```

npm does not complain during installation which made me hope that react-moment would work with the new React version.
Installed versions at time of compilation:

├── @types/node@17.0.38
├── @types/react-dom@18.0.5
├── @types/react@18.0.10
├── moment-timezone@0.5.34
├── moment@2.29.3
├── react-dom@18.1.0
├── react-moment@1.1.2
├── react@18.1.0
└── typescript@4.7.2

UPDATE: found this PR from 2018 which renames `React.SFC` to `React.StatelessComponent` and `React.FunctionComponent` https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30364

Thank you!

**Comment by AngelAlexQC** [comment]
I Hope that the creator accept my PR

**Comment by Stusu** [comment]
Hi @headzoo, any chance that it will be merged and released soon?

**Comment by headzoo** [comment]
@Stusu All set, published as v1.1.3.
